### ISSUE TITLE
Stampit not working with capybara/poltergeist

### ISSUE THREAD

**Issue by gabriel-dehan** [brettimus/stampitjs-rails#1]
Hello !

When doing integrations tests with capybara/poltergeist I get this error : 

``` javascript
TypeError: 'undefined' is not a function (evaluating 'shortcutMethod.bind(null, addMethods)')
```

It's about:
https://github.com/brettimus/stampitjs-rails/blob/master/vendor/assets/javascripts/stampit.js#L427

Could not find any reason for it. Any idea ?

**Comment by brettimus** [comment]
Hey! Thanks for opening an issue. I'm not sure what the culprit is, but I'll remind myself to look into this tomorrow!

—

**Comment by koresar** [comment]
We've seen this error before. Here is the recipe.
https://github.com/stampit-org/stampit/issues/175#issuecomment-149224759

TL;DR: The problem is PhantomJS is missing Function.bind. Include the [polyfill](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind#Polyfill) and it will work.

**Comment by brettimus** [comment]
@koresar thanks for digging that up! 

@gabriel-dehan let me know if that fixes your issue and I'll close

**Comment by gabriel-dehan** [comment]
Wow, perfect, thanks a lot. 

For future reference, just putting the polyfill in any of your JS files does not work you need to inject it into poltergeist, see http://blog.jachobsen.com/2013/10/17/poltergeist-phantomjs-and-javascript-injection/
